arch/inst/Zicond/czero.eqz.yaml

```diff
@@ -24,12 +27,21 @@ access:
   vu: always
 data_independent_timing: false
 operation(): |
+  XReg input1 = X[rs1];
+  XReg input2 = X[rs2];
+  XReg output = 0;
+
+  if(input2 != 0) {
+    output = input1;
+  }
+
+  X[rd] = output;
 
 sail(): |
   {
     let value = X(rs1);
     let condition = X(rs2);
-    let result : xlenbits = if (condition != zeros()) then zeros()
+    let result : xlenbits = if (condition == zeros()) then zeros()
```

Comment:
Good catch! You should mention that you are fixing this bug in your commit message and PR.

I also opened an issue with the project that produced this content: https://github.com/ThinkOpenly/sail/issues/49.